lib/middleware.js

```diff
@@ -73,7 +73,7 @@ exports.notFound = function () {
 
 exports.error = function (options) {
   options = options || {};
-  const properties = ['status', 'message'];
+  const properties = ['status'];
```

Comment:
instead of removing message from here and setting explicitly in LN92. Can you rather add both message and reason in to this array and avoid LN 92 & 93?

Reply:
According to OSB API spec, the broker error should have description field, not message. https://github.com/openservicebrokerapi/servicebroker/blob/v2.13/spec.md#broker-errors

And for errors like Unprocessable Entity, the attribute is 'error', as defined here https://github.com/openservicebrokerapi/servicebroker/blob/v2.13/spec.md#asynchronous-operations

Hence, setting it explicitly.

Reply:
Ok. Got it. Thanks!